app/src/main/java/org/mozilla/focus/utils/DialogUtils.java

```diff
@@ -206,6 +206,59 @@ public static void showPrivacyPolicyUpdateNotification(Context context) {
         NewFeatureNotice.getInstance(context).setPrivacyPolicyUpdateNoticeDidShow();
     }
 
+    public static Dialog showSpotlight(@NonNull final Activity activity, @NonNull final View targetView, @NonNull DialogInterface.OnCancelListener onCancelListener, int messageId){
```

Comment:
Can we re-use the code in showMyShotOnBoarding() and rename it to showSpotlight?

Reply:
I think it would be better if we separate the reuse clean up to another patch